requirements/base.in

```diff
@@ -1,4 +1,5 @@
 # Pure python dependencies
+elasticsearch-dsl
```

Comment:
```suggestion
elasticsearch-dsl~=8.0
```
I think this is the syntax to restrict it to the major version 8 - which should match the version of ES used :)